### PR TITLE
Update minimum-requirements.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more detailed information to Licensing requirements

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9325

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/minimum-requirements.md [hunk 1]
```diff
@@ -38,12 +38,13 @@ There are some minimum requirements for onboarding devices to the service. Learn
 
 ## Licensing requirements
 
+The standalone versions of [Defender for Endpoint Plan 1 and Plan 2](defender-endpoint-plan-1-2.md) do not include server licenses. To onboard servers to those plans, you'll need Defender for Servers Plan 1 or Plan 2 as part of the [Defender for Cloud](/azure/defender-for-cloud/defender-for-cloud-introduction) offering. To learn more. see Overview of [Microsoft Defender for Servers](/azure/defender-for-cloud/defender-for-servers-introduction).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/minimum-requirements.md]:
```suggestion
The standalone versions of [Defender for Endpoint Plan 1 and Plan 2](defender-endpoint-plan-1-2.md) do not include server licenses. To onboard servers to those plans, you'll need Defender for Servers Plan 1 or Plan 2 as part of the [Defender for Cloud](/azure/defender-for-cloud/defender-for-cloud-introduction) offering. To learn more, see the overview of [Microsoft Defender for Servers](/azure/defender-for-cloud/defender-for-servers-introduction).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**